### DISCUSSION TITLE
Is there a way to deactivate audit logs?

### DISCUSSION THREAD

**Discussion by idrikay** [gravitational/teleport#40076] [Q&A] [answered]
As the title suggests is there any way to deactivate the audit logs at least temporarily?

**Comment by stevenGravy** [comment, marked as answer]
No, you can turn off session recordings cluster wide or for some protocols by roles.

**Reply by idrikay** [reply]
Hi Steven, thank you for your answer. We have already removed session recordings but we were looking at audit logs specifically.
